Fail to compile egs_view in linux

**Describe the bug**
Hi there,

I’m trying to compile egs_view after a recent fresh installation/configuration using the shell script, as our department’s cluster now has Qt5 installed. However the compilation for egs_view fails.

I’ve properly set the QTDIR and QT_SELECT environment variables in my .bashrc file as recommended:

export QTDIR=/usr/lib64/qt5
export QT_SELECT=qt5


All other GUIs were successfully built using the same process within HEN_HOUSE/gui, so the issue seems specific to egs_view. I’d really appreciate any help identifying what might be causing this problem.

Thanks,

**To Reproduce**
A precise description of the steps to reproduce the behavior:

1. Within HEN_HOUSE/egs++/view type make clean and then make
2. An error message appears and compilation fails: 

viewcontrol.cpp: In constructor ‘GeometryViewControl::GeometryViewControl(QWidget*, const char*)’:
viewcontrol.cpp:92:25: error: array used as initializer
     : QMainWindow(parent) {

**Screenshots**

![Image](https://github.com/user-attachments/assets/35740aea-1db0-42e4-8781-316fbbcdb69b)

**Operating system**

- [x] Linux


**EGSnrc version**
v2025a

--- a/HEN_HOUSE/egs++/view/viewcontrol.h
+++ b/HEN_HOUSE/egs++/view/viewcontrol.h
@@ -211,7 +211,7 @@ private:
     bool showElectronTracks;
     bool showPositronTracks;
     bool hasDynamic; //boolean to track whether or not to make time objects visible or hidden
-    char head_inctime[20] = "include time index="; // must match the same in egs_particle_track.h
+    char head_inctime[20] = {'i','n','c','l','u','d','e',' ','t','i','m','e',' ','i','n','d','e','x','='}; // must match the same in egs_particle_track.h
     bool hasTrackTimeIndex;
     bool isPlaying;
     vector<bool> show_regions;

